core/templates/components/state-directives/hint-editor/hint-editor.component.spec.ts

```diff
@@ -118,11 +118,11 @@ describe('HintEditorComponent', () => {
 
   it('should check if hint HTML length exceeds 500 characters', () => {
```

Comment:
Checking the case having no p tags could fail tests unnecessarily, taking a scenario where a solution or a hint can be explained with an image, which is sufficient to explain, then it will fail and I feel full HTML can't be the issue for hint-editor because it will always return a Html string, I was having this issue with the service test as there was no mention about HTML strings, now we have explained every method with Jsdoc, so probability is less.
If I am assuming wrong then please let me know. Thanks!

Reply:
Yup, understood, thanks. I think I can see both sides here -- on the one hand, the function accepts non-HTML strings so we should check if that works correctly; on the other hand, that isn't, strictly speaking, part of its contract. I think overall the latter argument sounds reasonable, so it's fine to leave things as you have them here. Thanks!

Reply:
Done